plugins/modules/cloud/misc/terraform.py

```diff
@@ -32,6 +32,11 @@
       - The path to the root of the Terraform directory with the
         vars.tf/main.tf/etc to use.
     required: true
+  log_error_path:
+    description:
+      - The path to the root of the Terraform directory with the
+        vars.tf/main.tf/etc to use.
```

Comment:
This description looks wrong. You seem to have simply copied it. Please adjust it!

Reply:
Yeah, forgot that